### DISCUSSION TITLE
What type of car was "KITT" in the original Knight Rider TV show?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#677] [Galaxy Trivia] [answered]
**Bonus:** What does KITT stand for?

**Comment by SANKARAMDAS** [comment, marked as answer]
The KITTs are two distinct elements: one known as the "Knight Industries Two Thousand", which showed up in the first TV arrangement Knight Rider, and the different as the "Knight Industries Three Thousand", which showed up first in the two-hour 2008 pilot film for another Knight Rider TV arrangement and afterward the new arrangement itself.

Type - 1982 Pontiac Trans Am
In the original Knight Rider series, the character of KITT (Knight Industries Two Thousand) was physically embodied as a modified 1982 Pontiac Trans Am. KITT was designed by customizer Michael Scheffe.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/SANKARAMDAS?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉
